.github/CODEOWNERS

```diff
@@ -142,29 +150,36 @@
 # New structures team.
 # Question project.
 /core/controllers/question*.py @aks681
-/core/domain/question*.py  @aks681
+/core/domain/question*.py @aks681
 /core/storage/question/ @aks681
 /core/templates/dev/head/domain/question/ @aks681
 /core/templates/dev/head/pages/question_editor/ @aks681
 /core/templates/dev/head/pages/questions_list/ @aks681
 
+# Practice session project.
+/core/templates/dev/head/pages/practice_session/ @aks681
```

Comment:
Assuming this to be a part of the new structure project.

Reply:
Perhaps @vinitamurthi should be codeowner? Comes under questions.

Reply:
Sure, FYI - I am also adding /core/templates/dev/head/pages/question_player/ in #6620